mockatests/vhd/Makefile.am

```diff
@@ -0,0 +1,21 @@
+AM_CFLAGS  = -Wall
+AM_CFLAGS += -Werror
+AM_CFLAGS += -fprofile-dir=/tmp/coverage/blktap/mockatests/vhd -fprofile-arcs -ftest-coverage
+AM_CFLAGS += -Og -fno-inline-functions -g
+AM_CFLAGS += -Doff64_t=__off64_t
+
+AM_CPPFLAGS = -I$(top_srcdir)/include -I$(top_srcdir)/cbt -I../include
+
+check_PROGRAMS = test-vhd-util
+TESTS = test-vhd-util
+
+test_vhd_util_SOURCES = test-vhd-util.c test-vhd-util-snapshot.c vhd-wrappers.c
+test_vhd_util_LDFLAGS = $(top_srcdir)/vhd/lib/.libs/libvhd.a -lcmocka -luuid
```

Comment:
You can avoid this nastiness and replace with

```
test_vhd_util_LDFLAGS = $(top_srcdir)/vhd/lib/libvhd.la -lcmocka -luuid
test_vhd_util_LDFLAGS += -static-libtool-libs
```

Reply:
Fixed